connector/src/main/java/org/odpi/egeria/connectors/juxt/crux/repositoryconnector/CruxOMRSRepositoryConnector.java

```diff
@@ -161,10 +161,10 @@ public synchronized void start() throws ConnectorCheckedException {
                 Files.delete(Paths.get(configFile.getCanonicalPath()));
             }
             Map<Keyword, ?> details = cruxAPI.status();
-            log.debug(" ... node: {}", details);
-            log.debug(" ... luceneConfigured? {}", luceneConfigured);
-            log.debug(" ... luceneRegexes?    {}", luceneRegexes);
-            log.debug(" ... synchronousIndex? {}", synchronousIndex);
+            log.info("crux config-- node: {}", details);
+            log.info("crux config -- luceneConfigured? {}", luceneConfigured);
+            log.info("crux config -- luceneRegexes?    {}", luceneRegexes);
+            log.info("crux config -- synchronousIndex? {}", synchronousIndex);
```

Comment:
The error handling here does not seem quite right - the audit log will output the stack trace if auditlog.logException() is called so the comment is not correct to say that it swallows the stack trace.  This means the reason for using log.error instead is insufficient.

The calls to the audit log do not include any properties that describe the values that gnerated the error.